service-deployment/src/cleaning-image/start.sh

```diff
@@ -24,9 +24,9 @@ if [ -d "/mnt/hdfs" ]; then
 
 fi
 
-if [ -d "/mnt/sdb1" ]; then
+if [ -d "/mnt/launcherlogs" ]; then
```

Comment:
>mnt [](start = 10, length = 3)

just to confirm: we don't need to change /mnt to a template because it is defined in cleaning.yaml.template (volumnMounts), right?

Reply:
The folder in container won't be affected.